ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-tech-preview.yaml

```diff
@@ -0,0 +1,79 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: elasticsearch-proxy
+promotion:
+  name: tech-preview
+  namespace: logging
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/es-proxy-logs ARTIFACT_DIR=${ARTIFACT_DIR}/es-proxy-artifacts make test
+  container:
+    from: src
+- as: cluster-logging-operator-e2e
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm
+      from: stable:cluster-logging-operator-e2e
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-aws
+- as: elasticsearch-operator-e2e
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm
+      from: stable:elasticsearch-operator-src
```

Comment:
Same here, correct to assume `stable` or need for `tech-preview` EO branch here?

Reply:
same